src/Particular.Analyzers/DiagnosticIds.cs

```diff
@@ -23,5 +23,7 @@ public static class DiagnosticIds
         public const string ImproperTryCatchSystemException = "PS0019";
         public const string ImproperTryCatchOperationCanceled = "PS0020";
         public const string MultipleCancellationTokensInATry = "PS0021";
+        public const string DateTimeAssignedToDateTimeOffset = "PS0022";
+        public const string UseUtcNow = "PS0023";
```

Comment:
Nitpick: the other analyzers are named in terms of the problem, rather than the solution, suggesting that this one should be something like:

```suggestion
        public const string NowUsedInsteadOfUtcNow = "PS0023";
```